lib/server.js

```diff
@@ -327,6 +328,28 @@ function create(log, error, config, routes, db, translator) {
     }
   )
 
+  // configure Sentry
+  const sentryDsn = config.sentryDsn
+  if (sentryDsn) {
+    Raven.config(sentryDsn, {})
+    server.on('request-error', function (request, err) {
+      let exception = ''
+      if (err && err.stack) {
+        try {
+          exception = err.stack.split('\n')[0]
```

Comment:
What are the changes of this containing sensitive information or PII, such as email addresses?

Reply:
@rfk Sentry got a built in scrubber if it detects PII (we cannot even disable it without devops). in our tests the first stack didn't have PII.

Reply:
Awesome, thanks. For completeness I'd like to get explicit confirmation from @jrgm or @jbuck that Travis is on board with us using this, but 👍 from me (especially since we already use it on the front-end).

Reply:
Also if it is easier we can skip including this stack information

Reply:
I like the stack a lot, just want to make sure we do our due diligence :-)